docker-compose.yml

```diff
@@ -7,6 +7,14 @@ services:
     volumes:
       - 'data:/data'
     network_mode: host
+    environment:
+      - 'ROUTER_INTERFACE=eth0'
+      - 'ROUTED_PREFIX=2001:db8:1::/64'
+      - 'TUNNEL_PREFIX=2001:db8:2::/64'
+      - 'TUNNEL_REMOTE_IP4=203.0.113.1'
+      - 'TUNNEL_LOCAL_IP4=192.0.2.1'
+      - 'TUNNEL_MTU=1480'
+      - 'CLIENTS_WHITELIST=2001:db8:1::'
```

Comment:
Is this `environment` section in `docker-compose.yml` needed? Default values are already being set in `start.sh`, and a single source of truth (for default values) is preferable. I assume (could be wrong) that the supervisor will set the environment variables defined in the web dashboard so they are visible in the app container, even if the variables are not listed in `docker-compose.yml`. I understand (could be wrong) that the only purpose of setting variables in `docker-compose.yml` is to set default values in case the variables are not defined in the web dashboard.